### PR TITLE
Install the qmk_udev helper program qmk_id

### PR DESCRIPTION
**Pull request by skullydazed**
## Description

Use the `qmk_id` program to identify keyboards. This way we don't change permissions for other rawhid devices.

### CODE REVIEW

**Comment by skullydazed** [comment]
@t-8ch Does this look correct to you?

File: build_deb_package.sh [hunk 1]
```diff
@@ -19,8 +19,12 @@ QMK_VERSION=$(qmk/usr/share/python/qmk/bin/qmk --version)
 virtualenv-tools --update-path /usr/share/python/qmk qmk/usr/share/python/qmk
 ln -s ../share/python/qmk/bin/qmk qmk/usr/bin/qmk
 
-# Copy in some other files
-cp 50-qmk.rules qmk/usr/lib/udev/rules.d/
+# Install the qmk_udev package
+git clone https://github.com/qmk/qmk_udev.git
+cd qmk_udev
+make
```
Comment (t-8ch) [inline build_deb_package.sh:25]: `make install` below will also build and test the binaries.

**Comment by t-8ch** [comment]
@skullydazed Except for one minor nitpick this looks good to me.

File: build_deb_package.sh [hunk 2]
```diff
@@ -19,8 +19,12 @@ QMK_VERSION=$(qmk/usr/share/python/qmk/bin/qmk --version)
...
+make DESTDIR=../qmk PREFIX=/usr install
```
Comment (skullydazed) [inline build_deb_package.sh:26]:
```suggestion
cd qmk_udev
make DESTDIR=../qmk PREFIX=/usr install
```

File: build_fedora_package.sh [hunk 3]
```diff
@@ -21,8 +21,14 @@ git commit -m"Update RPM version to ${QMK_VERSION}" README.md || true
 virtualenv-tools --update-path /usr/share/python/qmk qmk/usr/share/python/qmk
 ln -s ../share/python/qmk/bin/qmk qmk/usr/bin/qmk
 
+# Install the qmk_udev package
+git clone https://github.com/qmk/qmk_udev.git
+cd qmk_udev
+make
```
Comment (t-8ch) [inline build_fedora_package.sh:27]:
```suggestion
```

**Review by t-8ch — APPROVED** [review]

**Pull request merged**